Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -560,4 +560,46 @@ instance [Algebra R A] [FaithfulSMul R A] : FaithfulSMul R (FractionRing A) := b
   exact (FaithfulSMul.algebraMap_injective A (FractionRing A)).comp
     (FaithfulSMul.algebraMap_injective R A)
 
+section FaithfulSMul
+
+variable {R A} [Algebra R A] [FaithfulSMul R A]
+
+-- TODO: should this be stated for IsFractionRing also? But that would require the Algebra below
+
+noncomputable
+instance : Algebra (FractionRing R) (FractionRing A) :=
+  (IsFractionRing.map (FaithfulSMul.algebraMap_injective R A)).toAlgebra
```

Comment:
You should be able to just use [FractionRing.liftAlgebra](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/FractionRing.html#FractionRing.liftAlgebra)
```suggestion
noncomputable instance : Algebra (FractionRing R) (FractionRing A) :=
  (IsFractionRing.map (FaithfulSMul.algebraMap_injective R A)).toAlgebra
```